components/main_menu/main_menu.jsx

```diff
@@ -283,11 +284,11 @@ class MainMenu extends React.PureComponent {
                 <Menu.Group>
                     <TeamPermissionGate
                         teamId={this.props.teamId}
-                        permissions={[Permissions.MANAGE_SLASH_COMMANDS, Permissions.MANAGE_OAUTH, Permissions.MANAGE_INCOMING_WEBHOOKS, Permissions.MANAGE_OUTGOING_WEBHOOKS]}
+                        permissions={[Permissions.MANAGE_SLASH_COMMANDS, Permissions.MANAGE_OAUTH, Permissions.MANAGE_INCOMING_WEBHOOKS, Permissions.MANAGE_OUTGOING_WEBHOOKS, Permissions.MANAGE_SYSTEM]}
```

Comment:
@crspeller, added you specifically as a reviewer in case this was "known behaviour" and by design, given the dependency on the coarser `MANAGE_SYSTEM` permission for bots.

Reply:
There are more specific permissions for bots. Why are we falling back to manage system? Maybe there is something I am forgetting.

Reply:
Ok, I revisited this. The display of `Integrations` on the main menu now combines a team permission check for the existing bits + a system permission check for the bot bits, but relies exclusively on the bot permission bit instead of the bot account creation configuration flag since we always want to show the integrations page when you have permissions to manage bots, even if their creation is disabled.

I also fixed the backstage sidebar to reflect these semantics.